packages/react-instantsearch-dom/src/core/createInstantSearchServer.js

```diff
@@ -1,4 +1,4 @@
-import { isEmpty } from 'lodash';
+import { isEmpty, zipWith } from 'lodash';
```

Comment:
I think this is the only place in the code base using `zipWith`, so it would increase bundle size I think